docs/generate.php

```diff
@@ -97,9 +98,11 @@ class BuilderParser
         'setConfigurations',
         'generate',
         'generateAsync',
-        'getMultipartFormData',
         'fileName',
         'processor',
+        'type',
```

Comment:
I think it's not needed since this method has been removed in favor of PHP attribute.